src/main/java/com/bina/varsim/types/VCFInfo.java

```diff
@@ -0,0 +1,133 @@
+         */
+        public VCFInfoElement(String id, String value) throws UnexpectedException {
+            this.type = getType(id);
+            String[] valueArray = value.split(",");
+            switch(this.type) {
+                case "Integer":
+                    numberFields = new int[valueArray.length];
+                    for (int i = 0; i < valueArray.length; i++) {
+                        numberFields[i] = Integer.parseInt(valueArray[i]);
+                    }
+                    break;
+                case "String":
+                    stringFields = valueArray;
+                    break;
+                default:
+                    throw new UnexpectedException("ERROR: only Integer and String supported for INFO field (" + id + ").");
+            }
+        }
+
+        /**
+         * store id as a boolean field
+         */
+        public VCFInfoElement() {
+            this.type = "Boolean";
+            this.flagValue = true;
+        }
+
+        /**
+         * return appropriate values based on types
+         * @return return should be casted
+         */
+        public Object getValue() {
+            switch (this.type) {
+                case "Integer":
+                    return this.numberFields;
+                case "String":
+                    return this.stringFields;
+                case "Boolean":
+                    return this.flagValue;
+                default:
+                    return null;
+            }
+        }
+    }
+
+    /**
+     * return hard-coded type for some INFO IDs (including some reserved IDs in VCF
+     * spec)
+     * TODO: replace hard-coded infoID-type mapping with VCF header defined mapping
+     *
+     "##INFO=<ID=SVLEN,Number=A,Type=Integer,Description=\"Length of variant\">\n" +
+     "##INFO=<ID=POS2,Number=A,Type=Integer,Description=\"1-based Start position of source sequence\">\n" +
+     "##INFO=<ID=END2,Number=A,Type=Integer,Description=\"1-based End position of source sequence\">\n" +
+     "##INFO=<ID=SVTYPE,Number=1,Type=String,Description=\"Type of structural variant\">\n" +
+     "##INFO=<ID=CHR2,Number=A,Type=String,Description=\"Chromosome of source sequence\">\n" +
+     "##INFO=<ID=TRASUBTYPE,Number=A,Type=String,Description=\"Subtype of translocation event:" +
+     " source sequence deleted (SELFISHNESS); source sequence accepted (CHIVALRY).\">\n"
+
+     * @param infoID
+     * @return
+     */
+    public static String getType(String infoID) {
+        if (infoID.equals("SVLEN") || infoID.equals("POS2") || infoID.equals("END2") || infoID.equals("END")
+                || infoID.equals("DP")) {
+            return "Integer";
+        } else if (infoID.equals("SVTYPE") || infoID.equals("CHR2") || infoID.equals("TRASUBTYPE")) {
+            return "String";
+        } else {
+            throw new IllegalArgumentException("ERROR: unrecognized INFO ID (" + infoID + ").");
+        }
+    }
+
+    public static void main(String[] args) {
```

Comment:
Since we are already doing tests separately, can consider moving this test there.